Fix:

implicit declaration of function 'ioctl' is invalid in C99
implicit declaration of function 'creat' is invalid in C99
implicit declaration of function 'open' is invalid in C99
--- correct.c.orig	2015-02-08 02:35:41.000000000 -0600
+++ correct.c	2020-10-26 01:14:28.000000000 -0500
@@ -209,6 +209,7 @@
 #include "proto.h"
 #include "msgs.h"
 #include "version.h"
+#include <fcntl.h>
 
 void		givehelp P ((int interactive));
 void		checkfile P ((void));
--- lookup.c.orig	2005-04-14 16:25:52.000000000 -0500
+++ lookup.c	2020-10-26 01:15:03.000000000 -0500
@@ -89,6 +89,7 @@
 #include "ispell.h"
 #include "proto.h"
 #include "msgs.h"
+#include <fcntl.h>
 
 int		linit P ((void));
 #ifdef INDEXDUMP
--- term.c.orig	2015-02-08 02:35:41.000000000 -0600
+++ term.c	2020-10-26 01:13:50.000000000 -0500
@@ -99,6 +99,7 @@
 #endif
 #endif
 #include <signal.h>
+#include <sys/ioctl.h>
 
 void		ierase P ((void));
 void		imove P ((int row, int col));
